llms/mlx_lm/tuner/utils.py

```diff
@@ -249,12 +249,13 @@ def remove_lora_layers(model: nn.Module) -> nn.Module:
     return model
 
 
-def print_trainable_parameters(model):
-    def nparams(m):
-        if isinstance(m, (nn.QuantizedLinear, nn.QuantizedEmbedding)):
-            return m.weight.size * (32 // m.bits)
-        return sum(v.size for _, v in tree_flatten(m.parameters()))
+def nparams(module):
+    if isinstance(module, (nn.QuantizedLinear, nn.QuantizedEmbedding)):
+        return module.weight.size * 32 // module.bits
+    return sum(v.size for _, v in tree_flatten(module.parameters()))
```

Comment:
Do you mind making two small modifications to this:

1. Include the [quantized switch layer](https://github.com/ml-explore/mlx-examples/blob/main/llms/mlx_lm/models/switch_layers.py#L9) in this list.
2. Include the bias if present (I think we need that right o/w under counting params)

Reply:
I switch it to check for the "bits" attribute. Hopefully that's not too brittle. Good call on the bias.

Reply:
SGTM! Thanks!